index.js

```diff
@@ -184,6 +205,69 @@ function addAttribute(props, prop, value, ctx) {
   }
 }
 
+function vueMergeAttributes(props, info, value) {
+  var groupProp = vueGroupAttribute(info, value)
+  var propName = groupProp.name
+  var propValue = groupProp.value
+
+  props[propName] =
+    typeof propValue === 'object'
+      ? Object.assign({}, props[propName] || {}, propValue)
+      : Array.isArray(propValue)
+        ? [].concat(props[propName] || [], propValue)
+        : propValue
+}
+
+function vueGroupAttribute(info, value) {
+  var name = info.property
+  var props = {}
+
+  if (vueUtils.isTopLevel(name)) {
+    props.name = name
```

Comment:
I feel like you’re setting `.name` and `.value` a lot. Maybe use variables for that, and only set these two at the end of the function?